src/sentry/static/sentry/app/views/organizationIntegrations/sentryAppInstallationDetail.tsx

```diff
@@ -33,38 +34,11 @@ class SentryAppInstallationDetail extends React.Component<Props> {
     app: PropTypes.object.isRequired,
     onAppUninstall: PropTypes.func.isRequired,
     onAppInstall: PropTypes.func.isRequired,
+    onCloseModal: PropTypes.func,
   };
 
   redirectUser = (install: SentryAppInstallation) => {
     const {organization, app} = this.props;
-
-    trackIntegrationEvent(
```

Comment:
The analytics events should have been in the caller function of `redirectUser` which is `handleInstall` since we should first call `integrations.installation_start` before the API call to create the installation. This is how we do it for first party integrations.